espanol/templo/templo.md

```diff
@@ -1,5 +1,4 @@
-Cuando estaban todos reunidos en las afueras del templo, el Consejero empezó a predicar como todas
-las tardes.
+Cuando estaban todos reunidos en las afueras del templo, el Consejero empezó a predicar como todas las tardes. De pronto...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.